scripts/lib/pb_core_ingester.rb

```diff
@@ -81,14 +83,17 @@ def ingest_xml_no_commit(xml)
       raise ValidationError.new(e)
     end
 
-    begin
-      @solr.add(pbcore.to_solr)
-    rescue => e
-      raise SolrError.new(e)
+    if @re.match(xml)
```

Comment:
@mccalluc an emtpy regex will match anything, but will it slow down ingest? Would it be worth it to add `if @re.empty? || @re.match(xml)` to prevent running the regex if it's empty?